v2.0/upgrade-cockroach-version.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Upgrade a Cluster's Version
+title: Upgrade to CockroachDB v2.0
```

Comment:
@tschottdorf, please check changes to this file, specifically under "Finalize the upgrade".

Reply:
Actually, @tschottdorf, I'm going to merge this to fix a broken link holding up other PRs. Please comment nonetheless and I'll open a follow-up PR with any changes you suggest.